### PR TITLE
ansible: create root udb user(s) from template

### PR DESCRIPTION
**Pull request by zopieux**

### CODE REVIEW

**Comment by Zopieux** [comment]
Oui j'ai testé, ça marche.

File: ansible/roles/udb/templates/prologin/roots.list [hunk 1]
```diff
@@ -0,0 +1 @@
+admin:admin
```
Comment (seirl) [inline ansible/roles/udb/templates/prologin/roots.list:1]: This should be in the ansible inventory, unless you want everyone to be able to root the infra with `admin:admin`

File: ansible/roles/udb/tasks/main.yml [hunk 2]
```diff
@@ -67,3 +67,34 @@
...
+    owner: udb
+    group: udb
+    mode: 0750
+
+- name: Import initial root users
+  django_manage:
+    command: >
+      batchimport
+        --file /opt/prologin/udb/roots.list --type root
+        --update --logins --passwords
```
Comment (seirl) [inline ansible/roles/udb/tasks/main.yml:93]: We should use a module for this, like I did for udb.

Comment (Zopieux) [reply ansible/roles/udb/tasks/main.yml:93]: Did you mean `s/udb/mdb/`? I see `ansible/library/mdb.py`, nothing about udb.

Comment (seirl) [reply ansible/roles/udb/tasks/main.yml:93]: Yes

**Comment by juli0z** [comment]
Migrated to https://gitlab.com/prologin/tech/infra/sadm/-/merge_requests/210

**Pull request closed without merging**